java/FoodTruckApp/src/main/java/com/my/FoodTruckApp/order/OrderService.java

```diff
@@ -1,101 +1,52 @@
 package com.my.FoodTruckApp.order;
 
+import com.my.FoodTruckApp.appetizer.Appetizer;
 import com.my.FoodTruckApp.appetizer.AppetizerRepository;
-import com.my.FoodTruckApp.appetizer.AppetizerService;
+import com.my.FoodTruckApp.entree.Entree;
 import com.my.FoodTruckApp.entree.EntreeRepository;
-import com.my.FoodTruckApp.entree.EntreeService;
-import com.my.FoodTruckApp.order.NewOrderRequestBody;
-import com.my.FoodTruckApp.order.Order;
-import com.my.FoodTruckApp.order.OrderRepository;
 import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
 import org.springframework.stereotype.Service;
-import org.springframework.web.bind.annotation.PathVariable;
 
 import java.util.ArrayList;
-import java.util.Optional;
+import java.util.List;
+import java.util.stream.Collectors;
 
-@Service //annotation that tells spring THIS IS OUR SERVICE
-@RequiredArgsConstructor //necessary for the dependency injection to the repository
+@Service
+@RequiredArgsConstructor
+@Slf4j
 public class OrderService {
 
-    private final EntreeService entreeService; //getting an instance of the service, THIS IS THE DEPENDENCY INJECTION
-    private final AppetizerService appetizerService; //spring will look through the variables and see if they have a DEPENDENCY on this service
-    private final EntreeRepository entreeRepository; //we only have to instantiate this once
-    private final AppetizerRepository appetizerRepository; //these are picked up by the @RequiredArgsConstructor
+    private final EntreeRepository entreeRepository;
+    private final AppetizerRepository appetizerRepository;
     private final OrderRepository orderRepository;
-//    private final JdbcRepository jdbcRepository;
-    // -----------------------------------------------------------------------------------------------------------------------------
 
-//    public String getListOfAppsAndEntrees() {
-//        ArrayList<Entree> entrees = entreeService.getListOfEntrees();
-//        ArrayList<Appetizer> appetizers = appetizerService.getListOfAppetizers();
-//        System.out.println("These are the entrees: " + entrees);
-//        System.out.println("These are the appetizers: " + appetizers);
-//        return "entrees + appetizers";
-//    }
-//    public ArrayList<Entree> gettingEntrees() {
-//        ArrayList<Entree> entrees = entreeRepository.getAllEntrees();
-//        System.out.println("These are the entrees: " + entrees);
-//        return entrees;
-//    }
     public ArrayList<Order> getListOfOrders() {
-        ArrayList<Order> orders = orderRepository.getAllOrders();
-        System.out.println("There are the orders: " + orders);
+        ArrayList<Order> orders = orderRepository.getListOfOrders();
         return orders;
     }
```

Comment:
Do you wanna return the result right the way, or do you need to do something else with it first?

Reply:
I do not need to do anything with it, it has been inlined